src/config/dynamic-theme-fixes.config

```
@@ -9667,6 +9667,15 @@ INVERT
 
 ================================
 
+publicwww.com
+
+CSS
+body {
+    background: var(--darkreader-neutral-background) !important; 
```

```suggestion
    background-image: none !important; 
```

Let's just remove the background-image.